src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: 1px;
+  width: $calendar-width;
+
+  @each $start-day, $margin-left in $days-of-week {
+    &--day-of-week_#{$start-day} > &__day--number_1 {
```

Comment:
Just use :nth-child for `calendar__day` class, no need to create lots on extra classes.